### ISSUE TITLE
Sequential with Keras has a training MSE near zero from the very first epoch with a large testing MSE. Increasing/decreasing dropout and early stopping has no effect (even drop out of 99)

### ISSUE THREAD

**Issue by woodrujm** [keras-team/keras#7929]
# df shape: 8155674 rows × 45 columns

input_dimi = len(df.columns) - 1

y = df[target_column].values
x = df.drop(target_column,axis=1).values

# Create Model
start = time()
early_stopping = EarlyStopping(monitor='val_loss', min_delta=0, patience=50, verbose=0, mode='auto')
model = Sequential()
model.add(Dense(input_dimi, input_dim=input_dimi, kernel_initializer='normal', activation='elu'))
model.add(Dropout(.9))
model.add(Dense(11, kernel_initializer='normal', activation='linear'))
model.add(Dense(1, kernel_initializer='normal'))
model.compile(loss='mean_squared_error', optimizer='adam')
tensorboard = TensorBoard(log_dir="logs/{}".format(time()))

# Fit and predict with model 
history = model.fit(x,y,validation_split=.33,epochs=500, batch_size=200,verbose=0,callbacks=[early_stopping, tensorboard])

# Plot Results
plt.plot(history.history['loss'])
plt.plot(history.history['val_loss'])
plt.title('model loss')
plt.ylabel('loss')
plt.xlabel('epoch')
plt.legend(['train', 'test'], loc='upper left')
plt.show()

end = time()
print(str((end-start)/3600) + " hours  =  " + str((end-start)/60) + " minutes  =  " +  str(end-start) + str(" seconds"))

![image](https://user-images.githubusercontent.com/25488611/30604692-f7a4c85a-9d28-11e7-8e8c-c59cd9fcac78.png)

**Comment by woodrujm** [comment]
p.s. I know this is difficult to understand without more information, really just wondering if anyone has run into a similar problem/ what their work around was

**Comment by woodrujm** [comment]
pps running with sigmoid as activation function fixes

**Comment by woodrujm** [comment]
There are a total of 8 millions rows. training a model on 5 million doesnt have this problem, once you use the full dataset it starts doing the above

**Comment by ymodak** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby
Thanks!

If you think I made a mistake, please open another issue.
